src/main/java/evm/TransferTest.java

```diff
@@ -0,0 +1,85 @@
+package evm;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.nio.charset.StandardCharsets;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+
+import org.bouncycastle.util.encoders.Hex;
+import org.ethereum.vm.DataWord;
+import org.ethereum.vm.client.*;
+import org.ethereum.vm.util.ByteArrayUtil;
+import org.ethereum.vm.util.HashUtil;
+import org.ethereum.vm.util.HexUtil;
+
+import static org.ethereum.vm.util.ByteArrayUtil.merge;
+
+
+public class TransferTest extends vmbase {
```

Comment:
tests should be implemented in a `src/test` package and not `src/main` package, also they should use JUNIT, see examples here: https://github.com/yhassanzadeh13/lightchain-container/blob/master/src/test/java/blockchain/LightChainNodeTest.java